openslides/users/main_menu.py

```diff
@@ -10,5 +10,5 @@ class UserMainMenuEntry(MainMenuEntry):
     verbose_name = ugettext_lazy('Users')
     required_permission = 'users.can_see_extra_data'
     default_weight = 50
-    pattern_name = 'user_list'
+    pattern_name = 'core_dashboard'
```

Comment:
This issue is about widgets but not about MainMenuEntries. Please add an issue about main menu entries.